src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -166,26 +178,14 @@ public double Percentage
             private set { SetAndRaise(PercentageProperty, ref _percentage, value); }
         }
 
-        public double IndeterminateStartingOffset
-        {
-            get => GetValue(IndeterminateStartingOffsetProperty);
-            set => SetValue(IndeterminateStartingOffsetProperty, value);
-        }
-
-        public double IndeterminateEndingOffset
-        {
-            get => GetValue(IndeterminateEndingOffsetProperty);
-            set => SetValue(IndeterminateEndingOffsetProperty, value);
-        }
-
         static ProgressBar()
         {
             ValueProperty.OverrideMetadata<ProgressBar>(new(defaultBindingMode: BindingMode.OneWay));
-            ValueProperty.Changed.AddClassHandler<ProgressBar>((x, e) => x.UpdateIndicatorWhenPropChanged(e));
-            MinimumProperty.Changed.AddClassHandler<ProgressBar>((x, e) => x.UpdateIndicatorWhenPropChanged(e));
-            MaximumProperty.Changed.AddClassHandler<ProgressBar>((x, e) => x.UpdateIndicatorWhenPropChanged(e));
-            IsIndeterminateProperty.Changed.AddClassHandler<ProgressBar>((x, e) => x.UpdateIndicatorWhenPropChanged(e));
-            OrientationProperty.Changed.AddClassHandler<ProgressBar>((x, e) => x.UpdateIndicatorWhenPropChanged(e));
+            ValueProperty.Changed.AddClassHandler<ProgressBar>((x, _) => x.UpdateIndicatorWhenPropChanged());
```

Comment:
Since this is a refactoring, in other places we are removing code like this and using an OnPropertyChanged override which is cleaner and faster. Since you are modifying this code it's probably a good idea to move to OnPropertyChanged.

Reply:
thanks! will do that!